Task/Tasks/TSKTask.h

```diff
@@ -243,6 +275,16 @@ extern NSString *const TSKTaskDidStartNotification;
 - (instancetype)initWithName:(NSString *)name NS_DESIGNATED_INITIALIZER;
 
 /*!
+ @abstract Returns the prerequisite keys that the receiver requires to run.
+ @discussion Returns an empty set by default. Subclasses can override this method to return a set of
+     keys for keyed prerequisites that the task requires. When the task is added to a workflow, the
+     workflow will ensure that the task’s required keyed prerequisites are fulfilled. If this property
+     is nil, no prerequisite keys are required.
```

Comment:
Will update to mention that subclasses should take care to include their superclass’s keys required keys.